apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html

```diff
@@ -76,9 +76,10 @@ <h2 class="userdashboard-profile__name">
                       <h2 class="fs-6">{% trans 'Initiator Organisations' %}</h2>
                       {% for organisation in request.user.organisations %}
                       <div>
-                          <a class="btn btn--link text-decoration-none text-muted d-flex align-items-center" href="{% url 'a4dashboard:project-list' organisation_slug=organisation.slug %}">
+                          <a class="btn btn--link text-decoration-none u-text--gray d-flex align-items-center" href="{% url 'a4dashboard:project-list' organisation_slug=organisation.slug %}">
                             {% if organisation.logo %}
                               <img
+                                class="icon__img"
                                 src="{% thumbnail organisation.logo '0x40' %}"
```

Comment:
`0x40` -> `0x32`
same for the other ones below

Reply:
if there is no icon the font awesome icon is still 40px, should that be changed as well?

Reply:
no i think that should be ok but will see on dev when there multiple orgs

Reply:
ok have to add new icon styling but all fixed now